notebook/bundlerextensions.py

```diff
@@ -0,0 +1,259 @@
+# Copyright (c) Jupyter Development Team.
```

Comment:
It might be nice to put this file inside notebook/bundler so it's all self contained.